assayist/processor/main_analyzer.py

```diff
@@ -0,0 +1,135 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from assayist.processor.base import Analyzer
+from assayist.processor.logging import log
+
+
+class MainAnalyzer(Analyzer):
+    """
+    Look at the json brew output and write neomodels for the basic items identified.
```

Comment:
Optional: `write neomodels for the basic items identified` => `add basic information to the database`